bin/ream/src/cli/mod.rs

```diff
@@ -33,6 +42,21 @@ pub struct NodeConfig {
     )]
     pub network: Arc<NetworkSpec>,
 
+    #[arg(long, help = "Set HTTP Port", default_value_t = DEFAULT_HTTP_PORT)]
+    pub http_port: u16,
+
+    #[arg(long, help = "Set the HTTP address", default_value_t = DEFAULT_HTTP_ADDRESS)]
+    pub http_address: IpAddr,
```

Comment:
```suggestion
    #[arg(long, help = "Set the HTTP address", default_value_t = DEFAULT_HTTP_ADDRESS)]
    pub http_address: IpAddr,

    #[arg(long, help = "Set HTTP Port", default_value_t = DEFAULT_HTTP_PORT)]
    pub http_port: u16,
```

I think these make more sense in this order